iron-label.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-label.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+
+declare namespace Polymer {
+
+  class IronLabel extends Polymer.Element {
+
+    /**
+     * An ID reference to another element that needs to be
+     * labelled by this `iron-label` element.
+     */
+    for: string|null|undefined;
+    _forElement: Element|null;
+    attached(): void;
+    ready(): void;
+
+    /**
+     * generate a unique id for this element
+     */
+    _generateLabelId(): void;
+    _findTarget(): any;
+    _tapHandler(ev: any): void;
+    _applyLabelledBy(): void;
+    _forChanged(): void;
+  }
+}
+
```

Comment:
Ignores `Polymer.IronLabel._labelNumber: number`

https://github.com/PolymerElements/iron-label/blob/master/iron-label.html#L142

Reply:
Hmm, is this intended to be part of the public API? The underscore implies not maybe, but I'm not sure.

I don't believe Analyzer has support for scanning plain old variables like this, so I'll add a manual typing for it, if we do want it.

Reply:
This issue is more important in iron meta; I saw that Polymer.IronMeta not being typed was breaking some elements from being typed by modulizer